### PR TITLE
Log rate limit

### PR DESCRIPTION
**Pull request by Benjamintf1**
Thank you for submitting a pull request to the rep repository!

We appreciate the contribution. To help us understand the context for your pull request, please fill out this template to the best of your ability.

## Please make sure to complete all of the following steps

1. Check the [Contributing document](https://github.com/cloudfoundry/diego-release/blob/develop/CONTRIBUTING.md) on how to sign the CLA and run tests.
1. Submit your PR to this repo.
1. [**Submit an accompanying PR Review Request**](https://github.com/cloudfoundry/diego-release/issues/new?assignees=&labels=&template=pr-review-request.md&title=%5BREP+PR+REVIEW%5D%3A) referencing this PR so the Diego Team knows to review your pull request. 
* **Note: this PR will not be reviewed unless you submit the [PR Review Request](https://github.com/cloudfoundry/diego-release/issues/new?assignees=&labels=&template=pr-review-request.md&title=%5BREP+PR+REVIEW%5D%3A)**.

***************************

### What is this change about?

Allow developers and operators to individually limit the logging rate of a application

### What problem it is trying to solve?

Apps log too much and cause downtime and or too much downstream costs

### What is the impact if the change is not made?

we cannot do log rate limiting

### How should this change be described in diego-release release notes?

should be combined with bbs and executor changes

### Please provide any contextual information.

https://github.com/cloudfoundry/bbs/pull/50
https://github.com/cloudfoundry/executor/pull/61

### CODE REVIEW

File: generator/internal/evacuation_lrp_processor.go [hunk 1]
```diff
@@ -151,8 +141,3 @@ func (p *evacuationLRPProcessor) evacuateClaimedLRPContainer(logger lager.Logger
 
 	p.containerDelegate.DeleteContainer(logger, lrpContainer.Container.Guid)
 }
-
-func writeToStream(streamer log_streamer.LogStreamer, msg string) {
-	fmt.Fprintf(streamer.Stdout(), msg)
-	streamer.Flush()
```
Comment (mariash) [inline generator/internal/evacuation_lrp_processor.go:157]: We don't need a flush?

Comment (Benjamintf1) [reply generator/internal/evacuation_lrp_processor.go:157]: No, we don't need to. We removed the usage of the log stream object entirely, it's entirely unnecessary, would also break log rate metrics, and shouldnt be log rate limited, so we're just directly using the metron agent object directly now.

Comment (Benjamintf1) [reply generator/internal/evacuation_lrp_processor.go:157]: We replaced this in a lot of places in executor too. Everywhere but health checks and that doesn't have the same metrics problem cause it's the same object.

**Comment by Benjamintf1** [comment]
test

**Pull request merged**